en/drafts/originals/text-mining-youtube-comments.md

```diff
@@ -421,21 +424,21 @@ wf_feature_plot <- textplot_scale1d(tmod_wf_all, margin = "features") +
 wf_feature_plot
 ```
 
-{% include figure.html filename="en-or-text-mining-youtube-comments-03.png" alt="Visualization of WordFish model showing relative placement of features (words) with outliers circled in red" caption="Figure 3: Visualization of WordFish model showing relative placement of features (words) with outliers circled in red" %}
+{% include figure.html filename="en-or-text-mining-youtube-comments-03.png" alt="Visualization of WordFish model showing relative placement of features (words) with outliers circled in red" caption="Figure 3. Visualization of WordFish model showing relative placement of features (words) with outliers circled in red" %}
 
-This visualization shows all of the words found in the corpus of comments. Note how the visualization is roughly symmetric around the vertical axis, and how some words are further "out" from the sloping sides of the model than others.  These conspicuously displayed words are the strongest indicators of what each pole of the scaled dimension (along the horizontal axis) represents.  
+This visualization shows every words found in the corpus of comments. Note how it is roughly symmetric around the vertical axis, and how some words are further out from the sloping sides of the model than others. These conspicuously displayed words are the strongest indicators of what each pole of the scaled dimension (along the horizontal axis) represents.  

```

Comment:
I found the descriptions of the visualizations (figure 3, 4 and 5) generally quite tricky to follow, and am still not completely sure what I'm supposed to understand from these graphs. 

Perhaps this is more of a personal struggle, but I wonder whether you might have the capacity to try rewording/extending some of the descriptive text, to help readers better understand what they are seeing.